pixl_imaging/src/pixl_imaging/_orthanc.py

```diff
@@ -212,6 +256,10 @@ async def raise_if_pending_jobs(self) -> None:
     def aet(self) -> str:
         return str(config("ORTHANC_RAW_AE_TITLE"))
 
-    async def send_existing_study_to_anon(self, resource_id: str) -> Any:
+    async def send_study_to_anon(self, resource_id: str) -> Any:
         """Send study to orthanc anon."""
-        return await self._post("/send-to-anon", data={"ResourceId": resource_id})
+        return await self._post(
+            "/send-to-anon",
+            data={"ResourceId": resource_id, "Synchronous": False},
+            timeout=self.dicom_timeout,
```

Comment:
changing of timeouts, but we will then also need to monitor the job to make sure it succeeds, like we do with the c-move of DICOM

```suggestion
            data={"ResourceId": resource_id, "Synchronous": False, "Timeout": self.dicom_timeout},
            timeout=self.http_timeout,
```